OrbitLinuxTracing/PerfEventOpen.h

```diff
@@ -66,11 +66,11 @@ static constexpr uint64_t SAMPLE_REGS_USER_ALL =
 // of the entire record the kernel is willing to return is (1u << 16u) - 8.
 // If we want the size we pass to coincide with the size we get, we need to pass
 // a lower value. For the current layout of perf_event_stack_sample, the maximum
-// size is 65312, but let's leave some extra room.
+// size is 65312.
 // TODO: As this amount of memory has to be copied from the ring buffer for each
 //  sample, this constant should be a parameters and should be made available in
 //  some setting.
-static constexpr uint16_t SAMPLE_STACK_USER_SIZE = 65000;
+static constexpr uint16_t SAMPLE_STACK_USER_SIZE = 4 * 1024;
```

Comment:
I think that we shouldn't change this, as in general we can have a lot of stack samples bigger than this. Rather, we should improve how we read the samples from the ring buffer in order to only allocate and copy the necessary data on our side, by reading dyn_size first.